SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005885	{#993399}{ol}[Magical] - [Holy]{/}{/}{nl}Summon goddess' wagon to attack enemies.
+SKILL_20151224_005886	Divide the name of the enemy into numbers and calculate a new number.
+SKILL_20151224_005887	Dive the name of the enemy and calculate a new number from the first and the last letter.
+SKILL_20151224_005888	If the enemy is hit with the amount of the calculated number, the enemy is hit by the multiplication of the number.
+SKILL_20151224_005889	Final attack damage: #{CaptionRatio}#%
+SKILL_20151224_005890	Decrease the level of the enemy depending on the calculated number.
+SKILL_20151224_005891	Additional enemy level reduction: #{CaptionRatio}#
+SKILL_20151224_005892	Convert at certain rate when hit.
+SKILL_20151224_005893	Removal arranged
+SKILL_20151224_005894	Companion ability improvement
+SKILL_20151224_005895	Block increase, attack and critical hit rate reduced
+SKILL_20151224_005896	Rate of blocking enemy attack increased.
+SKILL_20151224_005897	Reducing cooltime
+SKILL_20151224_005898	State of reduced magical defense
+SKILL_20151224_005899	Freeze enemy at fixed rate.
+SKILL_20151224_005900	Evocation.
+SKILL_20151224_005901	State of reduced spirit and intelligence.
+SKILL_20151224_005902	Penalty is eased during combat incapability while buff is active.
+SKILL_20151224_005903	State of increased strength.
+SKILL_20151224_005904	State of continuos STA reduction.
+SKILL_20151224_005905	Hide
+SKILL_20151224_005906	Hidden state
+SKILL_20151224_005907	State of craft time reduction.
+SKILL_20151224_005908	Cyclone: Resist
+SKILL_20151224_005909	Increase harmful effect resistance rate
+SKILL_20151224_005910	Sacrifice HP recovery to 0 and raise magic amplification.
+SKILL_20151224_005911	Increase intelligence
+SKILL_20151224_005912	Increase defense
+SKILL_20151224_005913	Shield reduced instead of HP when hit by enemies
+SKILL_20151224_005914	Increase critical hit rate
+SKILL_20151224_005915	Increased critical hit rate
+SKILL_20151224_005916	Increase physical attack
+SKILL_20151224_005917	Increased physical attack
+SKILL_20151224_005918	Increase fire resistance
+SKILL_20151224_005919	Increased fire resistance
+SKILL_20151224_005920	Increase ice resistance
+SKILL_20151224_005921	Increased ice resistance
+SKILL_20151224_005922	Increase poison resistance
+SKILL_20151224_005923	Increased poison resistance
+SKILL_20151224_005924	Increase earth resistance
+SKILL_20151224_005925	Increased earth resistance
+SKILL_20151224_005926	Increase lightning resistance
+SKILL_20151224_005927	Increased lightning resistance
+SKILL_20151224_005928	Increase magical defense
+SKILL_20151224_005929	Increase magical attack
+SKILL_20151224_005930	Increased magical attack
+SKILL_20151224_005931	Porapore
+SKILL_20151224_005932	When stacked 100 times, increase holy and dark element by 6% of magic defense for 10 seconds.
```

Comment:
Increases Holy and Dark property attacks equal to 6% magic defense for 10 seconds when stacked 100 times.